gensim/models/tfidfmodel.py

```diff
@@ -65,21 +71,43 @@ def resolve_weights(smartirs):
         doesn't fit the list of permissible values.
 
     """
+
```

Comment:
PEP8 / PEP257: no blank line between docstring and function body.

Reply:
Fixed in 5d1213ad.